downgrade

```diff
@@ -187,7 +211,7 @@ process_term() {
     choice=${candidates[0]}
   else
     if present_packages "$term" "${candidates[@]}"; then
-      choice=$(read_selection "${candidates[@]}")
+      choice=$(read_selection "${candidates[@]}") || gettext "Invalid choice"
```

Comment:
Did you test this behavior? I'm not sure `read_selection` exits non-zero for an invalid choice, or it just prints nothing. You might want to check for `[ -z "$choice" ]` instead. Also, should this error, or just print and continue?

Reply:
It prints nothing when there is an invalid choice. Modified `process_term` to check `choice` and output to stderr. Now, it will print `Invalid choice` and will continue back to `main` where it will print another error message `Unable to downgrade foo`. I think it is fine this way.